### PR TITLE
v78.0.1

### PR DESCRIPTION
**Pull request by yarneo**
# 78.0.1

In this patch release we have provided a ChipTextField example along with some Beta theming bug fixes.

### ActionSheet

* [Removing theming extensions dependency on themers (#6654)](https://github.com/material-components/material-components-ios/commit/c2f6293c67a450d1b185c69cfd4de40be8587cb5) (Cody Weaver)
* [Remove Beta test from Ready component (#6677)](https://github.com/material-components/material-components-ios/commit/96c8a5f6e8efda4b4e29b9848daa52706ae36247) (Cody Weaver)

### Dialogs

* [Fix button font in Theming Extension (#6638)](https://github.com/material-components/material-components-ios/commit/669737f710c4693f3c85c8c369978b45ef187a8b) (Robert Moore)

### Ripple

* [Add Ripple snapshot test target to BUILD file (#6698)](https://github.com/material-components/material-components-ios/commit/19ccf9ca3af70f82e1e266cd136330eb2593896a) (Andrew Overton)

## Multi-component changes

* [Fixing Snapshot Tests (#6618)](https://github.com/material-components/material-components-ios/commit/419d0cc9ca51d2f91e26d8e9fb614c847636e900) (Andrew Overton)
* [Use MDCChipTextFieldScrollView to replace leftView functionality for flexibility and extensibility (#6686)](https://github.com/material-components/material-components-ios/commit/6f26784f86ac9d3b1ee583bc6655e968b690fa54) (Wenyu Zhang)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,56 @@
...
+
+### Snackbar
+
+* [Use test Typography scheme (#6695)](https://github.com/material-components/material-components-ios/commit/b99025053836a0c97b6b453914f78d379d375f08) (Robert Moore)
+
+## Multi-component changes
+
+* [Add test schemes. (#6690)](https://github.com/material-components/material-components-ios/commit/f1b5318d5ea7aa6f327e6a22a13ed4362e79cd8d) (Robert Moore)
+* [Fixing Snapshot Tests (#6618)](https://github.com/material-components/material-components-ios/commit/419d0cc9ca51d2f91e26d8e9fb614c847636e900) (Andrew Overton)
+* [Remove Beta test from Ready component (#6677)](https://github.com/material-components/material-components-ios/commit/96c8a5f6e8efda4b4e29b9848daa52706ae36247) (Cody Weaver)
```
Comment (romoore) [inline CHANGELOG.md]: This should be under "ActionSheet"

Comment (codeman7) [reply CHANGELOG.md]: +1

**Review by codeman7 — APPROVED** [review]

**Review by jverkoey — APPROVED** [review]

**Comment by yarneo** [comment]
Apologies but this will need a re-review, had to revert 8 commits due to a failing `pod lib lint`. We were relying on a new private testing podspec in our `MaterialComponents.podspec`.

**Comment by romoore** [comment]
The follow-up work for the revert can be tracked in https://github.com/material-components/material-components-ios/issues/6712

**Review by codeman7 — APPROVED** [review]

**Pull request merged**